src/main/java/seedu/address/model/entry/Name.java

```diff
@@ -54,8 +54,8 @@ public String toString() {
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
-                || (other instanceof Name // instanceof handles nulls
-                        && this.fullName.equals(((Name) other).fullName)); // state check
+               || (other instanceof Name // instanceof handles nulls
```

Comment:
The line that starts with `&&` needs to be indented more because it's actually in an 'inner expression'. No?